### DISCUSSION TITLE
Deployment error - wrong url and website not loading correctly

### DISCUSSION THREAD

**Discussion by KelingZ** [alshedivat/al-folio#2074] [Q&A] [answered]
I am working on the [al-folio](https://github.com/alshedivat/al-folio) repo with jekyll theme to create a personal website. 

I tried the following just to check if the deployment works with the original code. I forked the al-folio to my github depository and rename it as kkztest.github.io. After cloning it to my local machine, I made the following changes 

1. in _config.yml, change url into https://kkztest.github.io and leave baseurl as empty
2. in _pages/about.md add "Hey I just want to test out the deployment process."
3. stage, commit and push

Next I went to settings, change branch into gh-pages. 

The deployment was successful.

But

1. the url is not http://[name].github.io but http://[GithubUserName].github.io/[name].github.io
I have looked up the FAQ and I have set the url and base url correctly in the _config.yml file. The url in my _config.yml file is set as http://[name].github.io and the baseurl is empty.

Does anyone know how to set the website name as http://[name].github.io? Did I miss anything?

2. the website is not displayed correctly. It does reflect my change in about.md, but the overall layout is not right. 
I have looked up the FAQ but Ctrl + F5 does not seem to do the magic to make my website look normal. In addition, all links are broken in the published site. 

Here is the website link: https://kelingz.github.io/kkztest.github.io/
Here is the link to the repo: https://github.com/KelingZ/kkztest.github.io

Does anyone know how to fix the problem? 

Please see below for some screenshots. 
I appreciate your help and thank you in advance.

<img width="1152" alt="Screenshot 2024-01-14 at 1 51 25 PM" src="https://github.com/alshedivat/al-folio/assets/46659820/3945e7c2-caaa-4fa9-b9d7-2ab334dfc574">

<img width="1084" alt="Screenshot 2024-01-14 at 1 50 37 PM" src="https://github.com/alshedivat/al-folio/assets/46659820/bd07bdc4-6d0c-49a8-b7ee-8d7ff4ab8620">

<img width="1178" alt="Screenshot 2024-01-14 at 1 52 22 PM" src="https://github.com/alshedivat/al-folio/assets/46659820/c3c2f8b9-26c1-4d9c-b3c2-f38af8b0ff9d">

<img width="1430" alt="Screenshot 2024-01-14 at 1 53 28 PM" src="https://github.com/alshedivat/al-folio/assets/46659820/068f26b0-4d32-4afd-8792-57593652cd37">

**Comment by george-gca** [comment, marked as answer]
You got it wrong. Citing the referred part in [INSTALL.md](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md):

> Create a new repository using this template. For this, click on [Use this template -> Create a new repository](https://github.com/new?template_name=al-folio&template_owner=alshedivat) above the file list. If you plan to upload your site to your-github-username.github.io, note that the name of your repository MUST BE your-github-username.github.io or your-github-orgname.github.io, as stated in the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).

This means that you repository **MUST BE** named with your **GitHub username**, meaning `kelingz.github.io`. You can't choose the name before `github.io` since this domain is owned by GitHub. You can only set the name after that (`your_user.github.io/`), like it is the case for the demo site and you can change that by changing `baseurl` in [_config.yml](https://github.com/alshedivat/al-folio/blob/master/_config.yml). If you want to have more control over the url you are going to use, check the [GitHub pages docs](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages).

**Reply by KelingZ** [reply]
Thanks a lot! It works!
